### DISCUSSION TITLE
Start Image after reboot system.

### DISCUSSION THREAD

**Discussion by shernandezp** [mediagis/nominatim-docker#277] [General]
Hello,

I'm sorry for asking this question, but I haven't gotten used to docker yet. I have successfully installed nominatim 4.0, it works as expected. I restarted my machine and tried to start the image again with the command:
docker run -it --rm \
  -e PBF_URL=https://download.geofabrik.de/europe/monaco-latest.osm.pbf \
  -e REPLICATION_URL=https://download.geofabrik.de/europe/monaco-updates/ \
  -p 8080:8080 \
  --name nominatim \
  mediagis/nominatim:4.0
Of course,  it removes the data, then downloads and installs everything again.
How do I start the image without reinstalling all the stuff again?
My system is Ubuntu 20.04

Thanks.

**Comment by leonardehrenfried** [comment]
The `--rm` argument deletes the container after shutting it down. If you remove that it will stay around even after stopping. You can then use `docker start` to start it again.

If you want more persistence, read up what docker volumes are. The readme has more info.
